src/olympia/api/tests/test_middleware.py

```diff
@@ -54,7 +54,7 @@ def test_settings(self):
         # Sadly, raven inserts 2 middlewares before, but luckily the ones it
         # automatically inserts not modify the response.
```

Comment:
Might want to update that comment which the other two middlewares are these days.